minesweepr.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Minesweepr Game</title>
+  <style>
+  table, td {
+    background: #DCDCDC;
+    border: 1px solid #808080;
+    border-collapse: collapse;
+    text-align: center;
+  }
+
+  td {
+    height: 20px;
+    width: 20px;
+  }
+  </style>
+</head>
+
+<body>
+  <table>
```

Comment:
For this table, is the idea that later it'll be generated with some JS for a configurable grid size?